packages/js/block-templates/src/utils/register-woo-block-type.ts

```diff
@@ -25,14 +29,13 @@ interface BlockRepresentation< T extends Record< string, object > > {
 }
 
 type UseEvaluationContext = ( context: Record< string, unknown > ) => {
-	getEvaluationContext: (
-		select: typeof WPSelect
-	) => Record< string, unknown >;
+	getEvaluationContext: ( select: SelectType ) => Record< string, unknown >;
 };
 
 function defaultUseEvaluationContext( context: Record< string, unknown > ) {
 	return {
-		getEvaluationContext: () => context,
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unused-vars
+		getEvaluationContext: ( select: any ) => context,
```

Comment:
The argument (select) here isn't used, is there a reason we need that here?

Reply:
Good catch, this is absolutely not needed!

Removed in bc29d98 and confirmed tests and builds are still passing, as expected.